salt/modules/suse_apache.py

```diff
@@ -19,7 +19,7 @@ def __virtual__():
     """
     Only load the module if apache is installed.
     """
-    if salt.utils.path.which("apache2ctl") and __grains__["os_family"] == "Suse":
+    if salt.utils.path.which("a2enmod") and salt.utils.path.which("a2dismod") and __grains__["os_family"] == "Suse":
```

Comment:
I think maybe better to check this way as we are using the same sources not just for Tumbleweed, but for SLES 15 and Leap 15.x:
```suggestion
    if (
        salt.utils.path.which("apache2ctl") or
        (salt.utils.path.which("a2enmod") and salt.utils.path.which("a2dismod"))
    ) and __grains__["os_family"] == "Suse":
```